botcity/datapool/datapool.py

```diff
@@ -0,0 +1,180 @@
+
+
+import json
+from dataclasses import dataclass
+
+import requests
+
+from botcity.datapool.entry import DataPoolEntry, StateEnum
+from botcity.maestro import BotMaestroSDK
+
+from .enums import ConsumptionPolicyEnum, TriggerEnum
+
+
+@dataclass
+class DataPool:
+    label: str
+    default_activity: str
+    consumption_policy: str = ConsumptionPolicyEnum.FIFO
+    trigger: str = TriggerEnum.ALWAYS
```

Comment:
To avoid issues we must switch to NEVER by default.
```suggestion
    trigger: str = TriggerEnum.NEVER
```